statsmodels/tsa/vector_ar/tests/test_coint.py

```diff
@@ -17,8 +17,7 @@
 
 current_path = os.path.dirname(os.path.abspath(__file__))
 dta_path = os.path.join(current_path, "Matlab_results", "test_coint.csv")
-with open(dta_path, "rb") as fd:
-    dta = np.genfromtxt(fd)
+dta = pd.read_csv(dta_path, header=None).values
```

Comment:
this csv file is space delimited
this works for me:
dta = pd.read_csv(dta_path, header=None, delimiter="\s+").values